ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -167,6 +167,12 @@ literals:
   - ON
   - OFF
   - UNKNOWN
+- supply_fan_run_mode:
+  - AUTO
+  - MANUAL
+- bypass_supply_fan_status:
```

Comment:
what is this, the bypass for the inverter? would this not be covered by the [`supply_fan_run_mode`](https://github.com/google/digitalbuildings/blob/e123967128248b896d03dbc24d468e7a27d066d5/ontology/yaml/resources/fields/telemetry_fields.yaml#L1513)?

or perhaps it is an inverter mode of some kind, since it seems like you are suggesting this is for the inverter itself. not sure how you would bypass a supply fan otherwise.

Reply:
This only bypasses the drive in case of failure and motor runs at full speed. This field shows whether the motor is running in VFD control or is bypassed. This is not covered by the supply_fan_run_mode.

Reply:
Updated